docs/design/README.md

```diff
@@ -2179,15 +2179,22 @@ Libraries from the current package are imported by omitting the package name.
 ```carbon
 // Import the "Vertex" library from the same package.
 import library "Vertex";
-// Import the default library from the same package.
-import library default;
 ```
 
 The `import library ...` syntax adds all the public top-level names within the
 given library to the top-level scope of the current file as
 [`private`](#name-visibility) names, and similarly for names in
 [namespaces](#namespaces).
 
+An `import` declaration is not permitted to omit both the package name and the
+library. As a special case, for packages other than the main package, the
+default library can be imported with the syntax `import library default;`:
```

Comment:
Why is it the main package and the default library? Why not the main package and main library, or default package and default library?